src/commands/build.stanza

```diff
@@ -106,10 +112,14 @@ Static Dependency Construction
 With the task dependencies, external programs can be used to build
 the artifacts needed to load functionality from a static C library.
 To support this, the 'build' command defines the environment
-variable 'SLM_BUILD_STATIC'. This environment variable is intended
+variable 'SLM_BUILD_STATIC=True'. This environment variable is intended
 to indicate to any of the task dependencies that a static library
 is needed (as opposed to a shared library, like a *.dll)
 
+Additionally, the 'build' command defines the environment variable
+'SLM_BUILD_DYNAMIC=False'. This is the same env-var used by the 'repl'
```

Comment:
Is it `SLM_BUILD_DYNAMIC` or `SLM_BUILD_DYNAMIC_LIB`?

Reply:
Fixed. Thanks!